src/features/profile/profileSlice.ts

```diff
@@ -0,0 +1,42 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { IProfileStateType } from '../../app/types';
+
+const userInitial = {
+  user: 'Chayuga',
+} as IProfileStateType;
+
+// Creating Actions for Repos
+export const fetchProfileAction = createAsyncThunk('profile/list', async (user: string, { rejectWithValue }) => {
+  try {
+    // make http call.
+    const { data } = await axios.get(`https://api.github.com/users/${user}`);
+    return data;
+  } catch (error) {
+    if (error instanceof Error && !error?.message) {
+      throw error;
+    }
+    return rejectWithValue(error instanceof Error && error?.message);
+  }
+});
+
+// slice for Profile
+const profileSlices = createSlice({
+  name: 'repos',
```

Comment:
Name correctly, please. I think this causes issues if it's named the same as the reposSlice.